internal/overlord/servstate/handlers.go

```diff
@@ -356,8 +363,12 @@ func (s *serviceData) startInternal() error {
 	done := make(chan struct{})
 	go func() {
 		waitErr := s.cmd.Wait()
+		childErr := s.waitProcessGroup()
+		if childErr != nil {
+			logger.Noticef("Cannot wait for children processes after main process exit: %v", childErr)
+		}
 		close(done)
-		err := s.exited(waitErr)
+		err = s.exited(waitErr)
```

Comment:
This should be `:=` here to avoid setting the outer `err` (though if we change `childErr :=` above to just `err := s.waitProcessGroup()` then we won't need it.

Reply:
This was a typo actually from trying out things, but I agree with having `childErr` be just `err`, so I changed that.